<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="vdc01hpd-0072" name="Testcase_B2" tests="3" failures="3" timestamp="22 May 2014 07:06:45 GMT" time="0.672" errors="0">
  <testcase name="test1" time="0.0" classname="com.Framework.suiteB.Testcase_B2">
    <failure type="java.lang.AssertionError" message="expected [XYZ] but found [XYZasdf]">
      <![CDATA[java.lang.AssertionError: expected [XYZ] but found [XYZasdf]
	at com.Framework.suiteB.Testcase_B2.test1(Testcase_B2.java:55)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase name="test1" time="0.0" classname="com.Framework.suiteB.Testcase_B2">
    <failure type="java.lang.AssertionError" message="expected [XYZ] but found [XYZasdf]">
      <![CDATA[java.lang.AssertionError: expected [XYZ] but found [XYZasdf]
	at com.Framework.suiteB.Testcase_B2.test1(Testcase_B2.java:55)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase name="test1" time="0.0" classname="com.Framework.suiteB.Testcase_B2">
    <failure type="java.lang.AssertionError" message="expected [XYZ] but found [XYZasdf]">
      <![CDATA[java.lang.AssertionError: expected [XYZ] but found [XYZasdf]
	at com.Framework.suiteB.Testcase_B2.test1(Testcase_B2.java:55)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
</testsuite> <!-- Testcase_B2 -->
